internal/processor/parser/app.go

```diff
@@ -55,34 +91,11 @@ func NewAppParser(
 }
 
 func (am *AppParser) updateCacheLoop() {
-	// If an app hasn't sent a metric in a while,
-	// assume that it's either been taken down or
-	// that the loggregator is routing it to a different nozzle and remove it from the cache
-	ticker := time.NewTicker(time.Duration(clearCacheDuration) * time.Minute)
+	ticker := time.NewTicker(time.Duration(am.grabInterval) * time.Minute)
```

Comment:
Ok so this ticker is now used to periodically refresh the cache, is that correct?
Let's add a comment to explain it.